src/coreclr/pal/src/init/pal.cpp

```diff
@@ -102,7 +101,8 @@ using namespace CorUnix;
 // necessary prototype here
 //
 
-extern "C" BOOL CRTInitStdStreams( void );
+extern "C" BOOL CRTInitStdStreams(void);
+extern "C" VOID NUMASupportCleanup(void);
```

Comment:
NUMASupport is not initialized by the PAL anymore, so PAL should not be cleaning it up either. NUMASupportCleanup call should be deleted from here.